CustomApps/lyrics-plus/Utils.js

```diff
@@ -67,33 +69,23 @@ class Utils {
 
 		const cjkMatch = rawLyrics.match(new RegExp(kanaRegex.source + "|" + /\p{Unified_Ideograph}/gu.source + "|" + hangulRegex.source, "gu"));
-		let kanaCount = 0;
-		let tradCount = 0;
-		let simpCount = 0;
 
 		if (!cjkMatch) return;
 
-		for (const character of cjkMatch) {
-			if (character.match(hangulRegex)) {
-				return "ko";
-			}
-			if (character.match(kanaRegex)) {
-				kanaCount++;
-			}
-			if (character.match(simpRegex)) {
-				simpCount++;
-			}
-			if (character.match(tradRegex)) {
-				tradCount++;
-			}
-		}
+		const kanaCount = cjkMatch.filter(glyph => kanaRegex.test(glyph)).length;
+		const simpCount = cjkMatch.filter(glyph => simpRegex.test(glyph)).length;
+		const tradCount = cjkMatch.filter(glyph => tradRegex.test(glyph)).length;
+		const hangulCount = cjkMatch.filter(glyph => hangulRegex.test(glyph)).length;
 
 		const kanaPercentage = kanaCount / cjkMatch.length;
+		const hangulPercentage = hangulCount / cjkMatch.length;
```

Comment:
You dont need to do hangul percentage since its regex will either be true or false.

Reply:
I purposefully want to count the percentage of the Hangul included of the lyrics because of a possibility that a Chinese/Japanese song which includes some Korean references in which the lyrics show it in Korean. I do not want to make the whole song mistakenly detected as Korean while it is Chinese/Japanese. Sure, it's a hypothetical situation but don't disregard it either.

Reply:
there are no Korean characters in either the Japanese or Chinese alphabet so this is simply unneeded, kyrie had already told me not to do this in one of my previous PRs.

Reply:
Oh. In the case because of Kyrie said so I guess I have to retract this.

Reply:
I will revert this tomorrow.